cmd/node/node.go

```diff
@@ -48,9 +52,11 @@ import (
 	timeCfg "github.com/spacemeshos/go-spacemesh/timesync/config"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
+
+	_ "net/http/pprof"
 )
 
-import _ "net/http/pprof" // import for memory and network profiling
+// import for memory and network profiling
```

Comment:
the comment should be on the same line as the import - it explains it

Reply:
note that this also causes linting to fail

Reply:
Thanks, I did move the comment back where it was. I think go-fmt moved the single **import** into the big import and didn't moved the comment.